internal/topo/transform/template.go

```diff
@@ -25,9 +25,15 @@
 	"text/template"
 )
 
-type TransFunc func(interface{}) ([]byte, bool, error)
+// TransFunc is the function to transform data
 
-func GenTransform(dt string, format string, schemaId string, delimiter string) (TransFunc, error) {
+// The second parameter indicates whether to select fields based on the fields property.
+// If it is false, then after the dataTemplate, output the result directly.
+// If it is true, then after the dataTemplate, select the fields based on the fields property.
+
+type TransFunc func(interface{}, bool) ([]byte, bool, error)
```

Comment:
TODO: Add a breaking change notification in release note and plugin development tutorial